packages/calcite-components/src/components/loader/loader.scss

```diff
@@ -11,140 +11,113 @@
 
 @import "../../assets/styles/animation";
 
-$stroke-width: 3;
-$loader-scale: 54;
-$loader-circumference: ($loader-scale - (2 * $stroke-width)) * 3.14159;
-
 @media (prefers-reduced-motion: reduce) {
   :root {
     --calcite-internal-duration-factor: 0;
   }
 }
 
 :host {
-  @apply relative mx-auto hidden items-center justify-center opacity-100;
```

Comment:
Was loader hidden by default previously? Does the displays e2e test need to be updated?